lib/rex/random_identifier/generator.rb

```diff
@@ -111,11 +112,27 @@ class ExhaustedSpaceError < StandardError; end
     ).freeze
   )
 
+  PHPOpts = DefaultOpts.merge(
+    prefix: '$',
```

Comment:
I have a feeling we might need to come back to this pattern again in the future; as I imagine we'd want to use the same rig for variables as classnames as well as variables, and it wouldn't be possible to use this current approach for that

i.e.

```
<<~EOF
class #{rig['class']} {
  public function __wake() {
    $#{rig['payload']} = '...';
  }
}
EOF
```

Also in some scenarios we might want to have the identifier without the `$` prefix, such as when accessing params in PHP - such as `$_GET['#{rig['injection_param']}'];` etc